src/schema/schema.graphql

```diff
@@ -0,0 +1,67 @@
+scalar Date
+
+type Author {
```

Comment:
IMP: Readup on "Node" type in graphql, that is what will be needed to have global identifiers which are unique